watcher/src/watcher.rs

```diff
@@ -197,7 +197,7 @@ impl Watcher {
             for (src_url, (block_index, block_data_result)) in url_to_block_data_result {
                 match block_data_result {
                     Ok(block_data) => {
-                        log::info!(
+                        log::debug!(
```

Comment:
:question: Want to make sure we don't want to batch up like mentioned in https://github.com/mobilecoinfoundation/mobilecoin/issues/2405#issuecomment-1432279302 and turning these _off_ for normal use case is desired.

Reply:
Batching is for later and this PR is for 5.0 release